src/sentry/static/sentry/app/components/forms/select2Field.jsx

```diff
@@ -11,29 +11,31 @@ export default class Select2Field extends InputField {
           onChange={this.onChange.bind(this)}
           disabled={this.props.disabled}
           value={this.state.value}>
-          {this.props.choices.map((choice) => {
-            return (
-              <option key={choice[0]}
-                      value={choice[0]}>{choice[1]}</option>
-            );
-          })}
+        <option key="" value="">&nbsp;</option>
```

Comment:
this might not be useful -- i had added it to try and fix some funky rendering issues, and im not 100% sure it even solved the issue